sdk/cpp/src/runner/runner.cpp

```diff
@@ -163,14 +163,18 @@ void Logger::publish(const Sink::Level level, const std::string& msg) const {
   }
 }
 
-std::string stringify(const Sink::Level& log_level) {
-  static const std::map<Sink::Level, std::string> names = {
-      {Sink::Level::Debug, "debug"},
-      {Sink::Level::Info, "info"},
-      {Sink::Level::Warn, "warning"},
-      {Sink::Level::Error, "error"},
-  };
-  return names.at(log_level);
+static std::string stringify(const Sink::Level& log_level) {
+  switch (log_level) {
+    case Sink::Level::Debug:
+      return "debug";
+    case Sink::Level::Info:
+      return "info";
+    case Sink::Level::Warn:
+      return "warning";
+    case Sink::Level::Error:
+      return "error";
+  }
+  return "";
```

Comment:
Shouldn't we throw `std::invalid_argument` instead?

Reply:
it's unreachable, but yeah, I agree that it's better to throw that exception in case of UB.